### PR TITLE
Added capabilites feature to stateful client

### PR DESCRIPTION
**Pull request by mgamis-msft**
# What
Added capabilities feature to stateful client

# Why
https://skype.visualstudio.com/SPOOL/_workitems/edit/3336592

# How Tested
Console.log capabilities properties in Rooms call for Presenter:
image
and Consumer:
image

# Process & policy checklist

- [ ] I have updated the project documentation to reflect my changes if necessary.
- [ ] I have read the [CONTRIBUTING](https://github.com/Azure/communication-ui-library/blob/main/CONTRIBUTING.md) documentation.

**Is this a breaking change?**

- [ ] This change causes current functionality to break.

### CODE REVIEW

File: packages/calling-stateful-client/src/CallClientState.ts [hunk 1]
```diff
@@ -133,6 +135,19 @@ export interface TranscriptionCallFeatureState {
...
+/**
+ * State only version of {@link @azure/communication-calling#CapabilitiesFeature}
+ *
+ * @beta
+ */
+export interface CapabilitiesFeatureState {
+  /**
+   * Proxy of {@link @azure/communication-calling#CapabilitiesFeature.capabilities}.
+   */
+  capabilities: ParticipantCapabilities;
```
Comment (dmceachernmsft) [inline packages/calling-stateful-client/src/CallClientState.ts:148]: Do we want to call this `localParticipantCapabilities`?

**Review by dmceachernmsft — APPROVED** [review]
Small naming question otherwise looks great!

File: common/config/babel/.babelrc.js [hunk 2]
```diff
@@ -66,15 +66,19 @@ process.env['COMMUNICATION_REACT_FLAVOR'] !== 'beta' &&
...
       // A list of in progress beta feature.
       // These features are still beta feature but "in progress"
       // causing a build failure, but they are ignored by the preprocessing step.
       inProgressFeatures: [
         // Demo feature. Used in live-documentation of conditional compilation.
         // Do not use in production code.
-        'in-progress-beta-feature-demo'
+        'in-progress-beta-feature-demo',
+        // Feature for capabilities
+        'capabilities'
```
Comment (alkwa-msft) [inline common/config/babel/.babelrc.js:81]: do we want to commit this change as  well?

**Review by alkwa-msft — APPROVED** [review]

File: packages/calling-stateful-client/src/CallClientState.ts (hunk 1 above)
Comment (dmceachernmsft) [reply packages/calling-stateful-client/src/CallClientState.ts:148]: I know, something just seems a little funky that the type has participant in the name but not the definition itself. was just thinking readability sake, not blocking 😊

**Pull request merged**